src/main/java/mate/academy/Main.java

```diff
@@ -1,16 +1,48 @@
 package mate.academy;
 
+import java.time.LocalDate;
+import java.time.LocalDateTime;
+import java.time.format.DateTimeFormatter;
+import mate.academy.lib.Injector;
+import mate.academy.model.CinemaHall;
 import mate.academy.model.Movie;
+import mate.academy.model.MovieSession;
+import mate.academy.service.CinemaHallService;
 import mate.academy.service.MovieService;
+import mate.academy.service.MovieSessionService;
 
 public class Main {
+    private static final Injector injector = Injector.getInstance("mate.academy");
+
     public static void main(String[] args) {
-        MovieService movieService = null;
+        MovieService movieService = (MovieService) injector.getInstance(MovieService.class);
 
         Movie fastAndFurious = new Movie("Fast and Furious");
         fastAndFurious.setDescription("An action film about street racing, heists, and spies.");
         movieService.add(fastAndFurious);
         System.out.println(movieService.get(fastAndFurious.getId()));
         movieService.getAll().forEach(System.out::println);
+
+        CinemaHallService cinemaHallService = (CinemaHallService) injector
+                .getInstance(CinemaHallService.class);
+        CinemaHall kyiv = new CinemaHall();
+        kyiv.setCapacity(100);
+        kyiv.setDescription("Kyiv cinema-hall");
+        cinemaHallService.add(kyiv);
+
+        MovieSession movieSession = new MovieSession();
+        movieSession.setMovie(movieService.get(fastAndFurious.getId()));
+        DateTimeFormatter formatter = DateTimeFormatter.ofPattern("yyyy-MM-dd'T'HH:mm:ss");
+        LocalDateTime showTime = LocalDateTime.parse("2007-12-03T10:15:30", formatter);
+        movieSession.setShowTime(showTime);
+        movieSession.setCinemaHall(cinemaHallService.get(kyiv.getId()));
+
+        MovieSessionService movieSessionService = (MovieSessionService) injector
+                .getInstance(MovieSessionService.class);
+        movieSessionService.add(movieSession);
+        System.out.println(movieSessionService.get(movieSession.getId()));
```

Comment:
Ensure that the `get()` method in `movieSessionService` returns an `Optional<MovieSession>` and handle it appropriately to avoid potential `NoSuchElementException`. Consider using `orElseThrow()` with a custom exception if the movie session is not found.